package.json

```diff
@@ -102,20 +104,22 @@
     "jest": "^20.0.4",
     "jest-react-native": "18.0.0",
     "lint-staged": "^3.4.1",
+    "prettier": "^1.4.2",
     "react-dom": "16.0.0-alpha.12",
     "react-storybooks-relay-container": "^1.0.0",
     "react-test-renderer": "16.0.0-alpha.12",
     "recursive-readdir-sync": "^1.0.6",
     "relay2ts": "^0.2.0",
     "ts-jest": "^20.0.3",
-    "tslint": "^4.5.0",
+    "tslint": "^5.4.3",
     "typescript": "^2.3.2"
   },
   "peerDependencies": {
     "babel-relay-plugin": "^0.9.0"
   },
   "lint-staged": {
     "*.@(ts|tsx)": [
+      "yarn prettier-write --",
```

Comment:
This can’t really _fail_ the process of making a commit, right? Because that’s something I would like to run on each commit, whereas something like linting can actually fail the commit.